system-parachains/asset-hubs/asset-hub-polkadot/src/staking/mod.rs

```diff
@@ -15,5 +15,34 @@
 // along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
 
 //! Staking related config of the Asset Hub.
+//!
+//! The large pallets have their config in a sub-module, the smaller ones are defined here.
 
+pub mod bags_thresholds;
 pub mod nom_pools;
+
+use crate::*;
+
+impl pallet_fast_unstake::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type Currency = Balances;
+	type BatchSize = frame_support::traits::ConstU32<16>;
+	type Deposit = frame_support::traits::ConstU128<{ UNITS }>;
```

Comment:
might need to be updated, since should cost less on a parachain

Reply:
Updated to 1 / 100 UNIT.